lib/screen/home/home_page.dart

```diff
@@ -353,6 +360,20 @@ class HomePageState extends State<HomePage>
       SliverToBoxAdapter(
         child: HeaderView(paddingTop: paddingTop),
       ),
+      SliverToBoxAdapter(
+        child: MultiValueListenableBuilder(
+          valueListenables: [
+            configurationService.showTvAppTip,
+            configurationService.showCreatePlaylistTip,
+            configurationService.showLinkOrImportTip,
+          ],
+          builder: (BuildContext context, List<dynamic> values, Widget? child) {
+            return CarouselWithIndicator(
+              items: _listTipcards(context),
```

Comment:
question: do we allow to show multiple tipcards in homepage in one time (or just one at a time)? I think no?

Reply:
yes, I confirmed with @Anais-bitmark and she designed the UI for this case